dev/image/Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN curl -fsSL https://dl.google.com/cloudsql/cloud_sql_proxy.linux.amd64 > $CLO
 
 ### Helm3 ###
 RUN mkdir -p /tmp/helm/ \
-    && curl -fsSL https://get.helm.sh/helm-v3.0.3-linux-amd64.tar.gz | tar -xzvC /tmp/helm/ --strip-components=1 \
+    && curl -fsSL https://get.helm.sh/helm-v3.5.2-linux-amd64.tar.gz | tar -xzvC /tmp/helm/ --strip-components=1 \
     && cp /tmp/helm/helm /usr/local/bin/helm \
     && cp /tmp/helm/helm /usr/local/bin/helm3 \
```

Comment:
As you are on here, what do you think about adding this here?
```
 && helm completion bash > /usr/share/bash-completion/completions/helm \
```
I use `helm template` from time to time in Gitpod and I'm always annoyed that helm completion is not activated. Sorry for hijacking this PR for this. Fell free to ignore. :rofl: